packages/dropdowns/src/elements/menu/Menu.spec.tsx

```diff
@@ -148,8 +148,22 @@ describe('Menu', () => {
     );
 
     await floating();
+    const trigger = getByRole('button');
+
+    // open menu with onClick
+    await user.click(trigger);
     const item = getByTestId('item-02');
+    // focus remains on trigger with mouseEvents
+    expect(trigger).toHaveFocus();
 
+    // close menu
+    await user.click(trigger);
+    expect(item).not.toBeVisible();
+
+    // open menu with keyboard
+    trigger.focus();
+    await user.keyboard(' ');
+    // focus is on the focused item associated with `defaultFocusedValue`
```

Comment:
https://github.com/zendeskgarden/react-containers/pull/660 changed the focus behavior when using `defaultFocusedValue`.